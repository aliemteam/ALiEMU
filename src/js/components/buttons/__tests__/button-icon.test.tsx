import React from 'react';
import renderer from 'react-test-renderer';

import IconButton from '../button-icon';

it('should render correctly', () => {
    const tree = renderer.create(
        <>
            <IconButton icon="accessible_forward" />
            <IconButton icon="done_outline" size={50} />
            <IconButton icon="pets" size={25} color="red" />
        </>,
    );
    expect(tree).toMatchSnapshot();
});
