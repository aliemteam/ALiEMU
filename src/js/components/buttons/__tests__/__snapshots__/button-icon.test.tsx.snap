// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
Array [
  <button
    className="button"
  >
    <i
      className="material-icons"
      style={
        Object {
          "color": undefined,
          "fontSize": 18,
        }
      }
    >
      accessible_forward
    </i>
  </button>,
  <button
    className="button"
  >
    <i
      className="material-icons"
      style={
        Object {
          "color": undefined,
          "fontSize": 50,
        }
      }
    >
      done_outline
    </i>
  </button>,
  <button
    className="button"
  >
    <i
      className="material-icons"
      style={
        Object {
          "color": "red",
          "fontSize": 25,
        }
      }
    >
      pets
    </i>
  </button>,
]
`;
